conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/100/package/StructureDefinition-hrex-consent.json

```diff
@@ -942,11 +942,11 @@
 					{
 						"code": "Reference",
 						"targetProfile": [
-							"http://hl7.org/fhir/StructureDefinition/Organization",
-							"http://hl7.org/fhir/StructureDefinition/Patient",
-							"http://hl7.org/fhir/StructureDefinition/Practitioner",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient|4.0.0",
 							"http://hl7.org/fhir/StructureDefinition/RelatedPerson",
-							"http://hl7.org/fhir/StructureDefinition/PractitionerRole"
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner|4.0.0",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitionerrole|4.0.0",
+							"http://hl7.org/fhir/us/core/StructureDefinition/us-core-organization|4.0.0"
```

Comment:
Please switch to 3.1.1
Also update the CHANGELOG

Reply:
done